src/test/regress/expected/multi_cluster_management.out

```diff
@@ -180,6 +180,35 @@ SELECT 1 FROM master_add_node('localhost', :worker_2_port);
 (1 row)
 
 UPDATE pg_dist_placement SET shardstate=1 WHERE groupid=:worker_2_group;
+-- when there is no primary we should get a pretty error
+UPDATE pg_dist_node SET noderole = 'secondary' WHERE nodeport=:worker_2_port;
+SELECT * FROM cluster_management_test;
+ERROR:  group 3 does not have a primary node
+DETAIL:  the primary node may be in the process of failing over
```

Comment:
Maybe "node group 3 does not have a primary", since it's not entirely clear what "group" refers to.

The DETAIL is perhaps a little premature.